content/configuration/plugins.md

```diff
@@ -23,5 +23,18 @@ plugins: [
 ```
 
 A more complex example, using multiple plugins, might look something like this:
-
+```js
+plugins: [
+  // build optimization plugins
+  new webpack.optimize.CommonsChunkPlugin({ name: 'vendor', filename: 'vendor-[hash].min.js' }),
```

Comment:
Please format all plugin options as follows (this style is used througout the documentation): 
```javascript
new webpack.optimize.CommonsChunkPlugin({
  name: 'vendor',
  filename: 'vendor-[hash].min.js'
}),
```